addons/website_event_track/models/event.py

```diff
@@ -99,6 +99,23 @@ def open_track_speakers_list(self, cr, uid, track_id, context=None):
             'type': 'ir.actions.act_window',
         }
 
+    @api.model
+    def create(self, vals):
+        res = super(event_track, self).create(vals)
+        if res.user_id.partner_id not in res.message_follower_ids:
+            res.message_subscribe_users([res.user_id.id])
+        if res.speaker_ids not in res.message_follower_ids:
```

Comment:
You should subscribe message.speaker_ids. Anyway message_subscribe already check for existing subscription.